incubator/external-dns/templates/deployment.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels: {{ include "labels" . | indent 4 }}
+spec:
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels: {{ include "labels" . | indent 8 }}
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{- if .Values.image.repository }}{{ .Values.image.repository }}/{{- end -}}{{.Values.image.name}}:{{ .Values.image.tag }}"
```

Comment:
I think `.Values.image.repository` doesn't need to be guarded. If someone has it on dockerhub she could just use `repository: docker.io`?